extras/nginx-mtls-auth/README.md

```diff
@@ -0,0 +1,39 @@
+# Sample of using securing Model Server with nginx
+
+Dockerfile and scripts in this directory are an example of using Nginx mTLS module to implement authentication and authorization of OpenVINO Model Server.
+
+This can secure both GRPC and REST endpoints. Model Server will present server certificate, and allow connection only from clients who perform full TLS handshake (successful client certificate authentication).
+
+WARNING: Those contain certificate generation automation for development and testing purposes. Do not use those in production - follow best practices of your organization. For ensuring "fast fail", certificates generated here will expire after a single day.
+
+WARNING: Only minimal nginx configuration is presented. Your are responsible for developing and using a secure configuration.
+
+## Quick Start
+
+1. Ensure you have `openvino/model_server` image available; either locally, or you have an Internet connection to download official image.
+2. Run `./build.sh` to build nginx image extra layer.
+3. In terminal 1, execute `./generate_certs_and_start_secure_model_server.sh` script. It will generate certificates (insecure, for testing only), download sample model and start the container.
```

Comment:
they are secure, it should be sufficient to write they are self-signed for testing purposes. Follow your organization process for requesting and generating the server and client certificates.

Reply:
it would be useful to split creating certificates and starting the container.

Reply:
Changed.